RedditSharp/ToolBoxUserNotes.cs

```diff
@@ -51,17 +51,18 @@ public static IEnumerable<TBUserNote> GetUserNotes(IWebAgent webAgent, string su
                     foreach (JToken note in x["ns"].Children())
                     {
 
-                        TBUserNote uNote = new TBUserNote();
-                        uNote.AppliesToUsername = user.Key;
-                        uNote.SubName = subName;
-                        uNote.SubmitterIndex = note["m"].Value<int>();
-                        uNote.Submitter = mods[uNote.SubmitterIndex];
-                        uNote.NoteTypeIndex = note["w"].Value<int>();
-                        uNote.NoteType = warnings[uNote.NoteTypeIndex];
-                        uNote.Message = note["n"].Value<string>();
-                        uNote.Timestamp = UnixTimeStamp.UnixTimeStampToDateTime(note["t"].Value<long>());
-                        uNote.Url = UnsquashLink(subName, note["l"].ValueOrDefault<string>());
-
+                        TBUserNote uNote = new TBUserNote()
```

Comment:
This change I do like and would merge. The rest of the `out` syntax sugar I'm not a big fan of.